src/appshell/view/preferences/importpreferencesmodel.h

```diff
@@ -54,7 +54,7 @@ class ImportPreferencesModel : public QObject, public async::Asyncable
     Q_PROPERTY(bool needUseDefaultFont READ needUseDefaultFont WRITE setNeedUseDefaultFont NOTIFY needUseDefaultFontChanged)
 
     Q_PROPERTY(int currentShortestNote READ currentShortestNote WRITE setCurrentShortestNote NOTIFY currentShortestNoteChanged)
-
+    Q_PROPERTY(bool roundTempo READ roundTempo WRITE setRoundTempo NOTIFY roundTempoChanged)
```

Comment:
Please keep the empty line after this one, for better readability (same for the other places in this file)

Reply:
Think I pushed all the new suggestions, let me know if the readability looks good for this!